ETC.tsv

```diff
@@ -4328,7 +4328,7 @@ ETC_20150317_004504	Device Canyon
 ETC_20150317_004505	Zinute-2: attracted Zinute to the extraordinary power: 32 / Zinute-2: attracted Zinute to the extraordinary power: 32 / Zinute-2: attracted Zinute to the extraordinary power: 32 / Sauga_s: attracted Sauga to the extraordinary power: 32 / Sauga_s: attracted Sauga to the extraordinary power: 32
 ETC_20150317_004506	Yo! Heave! ho!
 ETC_20150317_004507	The tree is too hard!
-ETC_20150317_004508	What can we do when the monster appears?
+ETC_20150317_004508	What should we do if a monster appears?
```

Comment:
This is incorrect, the original question is asking What CAN we do WHEN, not what SHOULD we do IF. Can and should are too separate words. In this case the word can is being used to ask what possible solutions can be used when the monster appears. Should would be implying if anything should be done. If also assumes that the monster may or may not appear, but thats also incorrect. The translation says when the monster appears which implies that it will appear. The original question should remain untouched.